code/game/machinery/doors/windowdoor.dm

```diff
@@ -68,10 +68,10 @@
 		close()
 	return
 
-/obj/machinery/door/window/CanPass(atom/movable/mover, turf/target, height=1.5, air_group = 0)
+/obj/machinery/door/window/Cross(atom/movable/mover, turf/target, height=1.5, air_group = 0)
```

Comment:
Check issues against this

https://github.com/d3athrow/vgstation13/issues/9862
https://github.com/d3athrow/vgstation13/issues/9861